common/meta/types/proto/meta.proto

```diff
@@ -48,6 +48,56 @@ message ExportedChunk {
   repeated string data = 10;
 }
 
+message WatchRequest {
+  // key is the key to register for watching.
+  string key = 1;
+
+  // key_end is the end of the range [key, key_end] to watch.
+  // if key_end is None, then watch only key.
+  optional string key_end = 2;
+
+  enum FilterType {
+    // recv all kind update event.
+    ALL = 0;    
+    // filter only update event.
+    UPDATE = 1;
+    // filter only delete event.
+    DELETE = 2;
+  }  
+  FilterType filter_type = 3;
+}
+
+message Event {
+  enum EventType {
+    Update = 0;
+    Delete = 1;
+  }
+
+  string key = 1;
+
+  EventType event = 2;
+
+  // current hold current value of key(if any)
+  optional bytes current = 3;
+
+  // prev value of key
+  optional bytes prev = 4;
+}
```

Comment:
We do not need `EventType` anymore.
`(prev.is_some(), current.is_some())` is enough to describe the event type.